### PR TITLE
feat: generate an openapi client using the generated openapi spec.

### PR DESCRIPTION
**Pull request by chirino**
* fixed lots of the handler annotations so that a better openapi spec is generated.

### CODE REVIEW

**Review by nerdalert — APPROVED** [review]
Ran it through ec2-qa, connectivity checks out.

Seeing missing CIDR fields in nexctl.
```
$ ./nexctl --host https://api.try.nexodus.127.0.0.1.nip.io --username admin --password floofykittens organization list
Organization ID                          NAME      IPV4 CIDR     IPV6 CIDR     DESCRIPTION
7e50d965-d32c-4c2c-8ed4-19508a05229d     admin                                 admin's organization
```

Fix that and I'm LGTM (or on a followup since it doesn't really break anything). Its a big patch, but now is the time to get those in. Ty.

File: internal/handlers/invitations.go [hunk 1]
```diff
@@ -12,6 +12,19 @@ import (
...
+// @Id           CreateInvitation
+// @Tags         Invitation
+// @Accepts		 json
+// @Produce      json
+// @Param        Invitation  body     models.AddInvitation  true  "Add Invitation"
+// @Success      201  {object}  models.Invitation
+// @Failure      400  {object}  models.BaseError
+// @Failure      404  {object}  models.BaseError
+// @Failure		 429  {object}  models.BaseError
+// @Router       /api/api/invitations [post]
```
[the remark this answers is not in the document]
Comment (chirino) [reply internal/handlers/invitations.go]: good catch.

**Comment by chirino** [comment]
@nerdalert I think it should be good now.

**Comment by nerdalert** [comment]
@chirino nice, LGTM

**Pull request merged**